src/monoflow.ts

```diff
@@ -29,21 +28,28 @@ export class Workflow<Z, A, B> {
 
   else<C>(fn: (error: Error) => C): Workflow<Z, B, unknown>  {
     // Make a higher-order function when `this._err` exists.
+    const sym = Symbol.for("monoflow.hasError");
```

Comment:
You could simplify this by using a string. Monoflow users won't see this object, so you don't need to hide the keys.

Reply:
that's true! fixed.